src/components/forging/index.js

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import { connect } from 'react-redux';
+import { updateForgedBlocks, updateForgingStats } from '../../actions/forging';
+import ForgingComponent from './forgingComponent';
+import { getForgedBlocks, getForgedStats } from '../../utils/api/forging';
 
-const Forging = () => (
-  <h1>Forging</h1>
-);
+const mapStateToProps = state => ({
+  account: state.account,
+  peers: state.peers,
+  statistics: state.forging.statistics,
+  forgedBlocks: state.forging.forgedBlocks,
+});
+
+const mapDispatchToProps = dispatch => ({
+  loadForgedBlocks: (activePeer, limit, offset, generatorPublicKey) => {
```

Comment:
To keep our code style consistent, Please follow the approach in login, transaction and account components by moving the Api call to the component consuming the data, and only dispatching the data here.